py_mind_memo/view.py

```
@@ -578,6 +580,30 @@ def on_toggle_reference_mode(self, event):
         self.render()
         return "break"
 
+    def on_insert_icon(self, event):
+        if self.editor.is_editing(): return
```

Ruff (0.15.7)

[error] Multiple statements on one line (colon)

(E701)